app-center-webapps/src/main/webapp/skin/less/app-center.less

```diff
@@ -485,230 +485,229 @@
+        margin-left: auto;
       }
-      .v-text-field__slot {
-        height: 48px!important;
-        &:focus {
-          outline: none!important;
-        }
+      .appPaginator {
+        padding-top: 30px;
       }
-      .v-input__slot {
-        height: 36px!important;
+      .appPaginator button {
+        background: @primaryColor;
+        border: 1px solid @primaryColor;
+        color: @baseBackgroundDefault;
+        padding: 5px 10px;
+        border-radius: 5px;
+        box-shadow: 0 0 3px #ff7c4061;
       }
-      input:focus {
-        outline: none!important;
+      .appPaginator button:hover {
+        background: transparent;
+        color: @primaryColor;
       }
-      i {
-        color: @greyColorLighten1Default!important;
+      .appSearch {
+        .v-input__prepend-inner {
+          margin-top: 6px!important;
+        }
+        .v-text-field {
+          padding-top: 0!important;
+          margin-top: 0!important;
+        }
+        .v-text-field__slot {
+          height: 48px!important;
+          &:focus {
+            outline: none!important;
+          }
+        }
+        .v-input__slot {
+          height: 36px!important;
+        }
+        input:focus {
+          outline: none!important;
+        }
+        i {
+          color: @greyColorLighten1Default!important;
+        }
       }
-    }
-    #appCenterAdminSetup {
-      align-self: center;
-      i {
-        color: @greyColorLighten1Default!important;
+      #appCenterAdminSetup {
+        align-self: center;
+        i {
+          color: @greyColorLighten1Default!important;
+        }
       }
     }
   }
 }
-.VuetifyApp .userFavoriteApplications {
-  padding: 14px;
+.VuetifyApp .userFavoriteApplicationsContainer {
   border-radius: 4px;
-  flex: 1 1 23%;
   background: @baseBackgroundDefault;
+  
+  .userFavoriteApplications {
+    padding: 14px;
 
-  .favoriteAppTitle {
-    width: 100%;
-    color: #6d6d6d;
-    font-size: 20px;
-    font-weight: 500;
-    letter-spacing: 1px;
-    padding-top: 25px;
-    padding-bottom: 35px;
-  }
-  .favoriteApplication {
-    border-bottom: 1px solid #efefef;
-    display: inline-flex;
-    margin-bottom: 15px;
-    width: 100%;
-    border-radius: 8px!important;
-    .favoriteAppImage {
-      .appImage {
-        box-shadow: 0 0 5px darkgrey;
-        object-fit: cover;
-        overflow: hidden;
-      }
-      .default {
-        border-radius: unset;
-        box-shadow: none;
-        margin-top: -53px;
-        margin-left: 20px;
-      }
+    .favoriteAppsTitle {
+      width: 100%;
+      color: @darkGrey;
+      font-size: 20px;
+      font-weight: 500;
+      letter-spacing: 1px;
+      padding-top: 25px;
+      margin-bottom: 35px;
     }
-    .favoriteAppUrl {
-      max-width: 130px;
-      h5 {
-        color: #797979;
-        font-size: 15px;
-        padding-left: 10px;
-        flex: 1;
+    .favoriteApplication {
+      border-bottom: 1px solid #efefef;
```

Comment:
Can we use a variable ?

Reply:
This color is used in platform-ui but no variable for it.